internal/services/mongodb/instance.go

```diff
@@ -460,6 +467,20 @@ func ResourceInstanceRead(ctx context.Context, d *schema.ResourceData, m any) di
 
 	_ = d.Set("settings", map[string]string{})
 
+	cert, err := mongodbAPI.GetInstanceCertificate(&mongodb.GetInstanceCertificateRequest{
+		Region:     region,
+		InstanceID: ID,
+	}, scw.WithContext(ctx))
+
+	var buf bytes.Buffer
+
+	if err == nil && cert != nil {
+		_, copyErr := io.Copy(&buf, cert.Content)
+		if copyErr == nil {
+			_ = d.Set("tls_certificate", buf.String())
+		}
+	}
+
```

Comment:
Errors are checked but silently ignored, if the certificate fetch or read fails, no diagnostic or return occurs.